### DISCUSSION TITLE
Query regarding remeshing

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/FEPX#63] [Q&A] [answered]
Hii!
How to perform remeshing? Can anyone please elaborate on the procedure step-wise?

**Comment by mkasemer** [comment, marked as answer]
@sushdash - This may be a bit in-depth. I do not think that there is an automated workflow for this yet. In general, after a simulation has finished, you will use Neper to re-mesh the deformed microstructure. From there, the question is how to map the variables to the new mesh. You will then restart the simulation using the new mesh.

FEPX will expect that the restart files (written in binary) contain the information for the new mesh. From there, there are questions regarding equilibrium (@rquey has some insight, as he has done this in the past).

@rquey - would you mind elaborating on this? I think this might require some work on our end.

**Reply by rquey** [reply]
The method to do this with Neper and FEPX is detailed in [this paper](https://neper.info/docs/neper-reference-paper.pdf). It was used in [this paper](https://doi.org/10.1016/j.jmps.2011.11.005) and [this paper](https://doi.org/10.1016/j.jmps.2015.07.014).

Since then, FEPX has gone through a major code overhaul and, although there is no reason why remeshing cannot be supported again, it would still need some work to get it back.

**Reply by sushdash** [reply]
Hii!
The first paper you are referring which contains the method of remeshing, I am unable to open that. Can you please send it to me through the mail (sushrita.dash1998@gmail.com)?

**Reply by rquey** [reply]
I've just fixed the link.
